test/test_integration_cluster.rb

```diff
@@ -2,102 +2,122 @@
 require_relative "helpers/integration"
 
 class TestIntegrationCluster < TestIntegration
-  parallelize_me!
-
-  DARWIN = !!RUBY_PLATFORM[/darwin/]
-
-  def setup
-    skip NO_FORK_MSG unless HAS_FORK
-    super
-  end
+  parallelize_me! unless Puma.jruby?
 
   def teardown
     return if skipped?
     super
   end
 
   def test_pre_existing_unix
-    skip UNIX_SKT_MSG unless UNIX_SKT_EXIST
+    skip UNIX_SKT_MSG unless HAS_UNIX
+    setup_puma bind: :unix, ctrl: :unix
```

Comment:
This reads like you're setting up a control server and then not using it. Is that correct?

Reply:
stop_server_wait calls `run_pumactl 'stop'`